MontyTRC89/TombEngine
# Unify DoTitle and DoLevel

Decopypastes `DoTitle` and `DoLevel` control routines and merges them into single `DoLevel` routine. Also refactors inconsistent underlying methods as well.

Additionally, allows to spawn "real" Lara in title flyby, which means she can be killed and can interact with room geometry and objects.

**Things to test**

1) Do new game / new game with level select / load game from title menu.
2) Do load game from in-game.
3) Exit game from title and from in-game.
4) Enter / exit inventory and try to use various items from it.
5) Check if any weird things are happening in any of these cases during gameplay (object activation, Lara controls, triggers).
6) Test how Lara works in title level (`Flow.EnableLaraInTitle(true)` command in Gameflow.lua)
